_posts/2016/2016-03-24-ecommerce-barcamp-jena.md

```diff
@@ -0,0 +1,56 @@
+---
+layout: post
+title: "Visiting the eCommerceCamp in Jena"
+date: "2016-03-24 09:03:24"
+icon: shopping-cart
+categories: events
+authors: ["Benjamin"]
+---
+
+The [eCommerceCamp](http://www.ecommerce-camp.de) is an annual [BarCamp](http://barcamp.org) that takes place in Jena, the capital of the eCommerce sphere in Germany, where a lot of the magic of online shopping started and still happens today. As always the highly affordable tickets for the unconference were sold out upfront and around 300 participants from near and far were expected. The orga team from [tritum](http://www.tritum.de) and [marmalade](http://www.marmalade.de) did a terrific job by creating an inspiring environment, which let everyone look forward to up-to-date topics about shop systems, productivity habits, email marketing, new technologies, customer satisfaction and team collaboration or even juggling with circus balls.
+
+## The participants
+
+The clientele consists mainly of UI and UX designers, frontend and backend developers, project managers, some marketing people and also a few dedicated testers from web agencies with 1 to 20 people, who typically work with self-contained content management systems like [Magento](https://magento.com), [OXID](http://www.oxid-esales.com), [Shopware](https://en.shopware.com), [TYPO3](https://typo3.org) or even [WordPress](https://wordpress.com). Attendees from the global eCommerce players are typically lesser represented, but this doesn't mean that local matadors like [Intershop](http://www.intershop.com) or ePages are not there as well.
+
+{% image blog/blog-ecommerce-barcamp-1.jpg %}{% endimage %}
+
+## The setting
+
```

Comment:
General comment: I'd skip the weekdays, as you don't give an info when the unconference took place, which is absolutely okay. Everyone who is interested for more, can click the link.